src/components/SwitchCheckbox/SwitchCheckbox.tsx

```diff
@@ -73,6 +74,8 @@ const SwitchCheckbox = (props: Props) => {
     <div className={theme.outerWrap}>
 
       <label htmlFor={names.falseRadio} className={theme.falseRadio}>
+        <VisuallyHidden>{switchType === 'trueFalse' ? 'False switch' : 'No switch' }</VisuallyHidden>
```

Comment:
@Farhan-emgage please remove "switch" from all these. Just the "True, False, Null, Yes, No" words are sufficient. Thanks

Reply:
@weegrom  The changes are done